libexec/scoop-update.ps1

```diff
@@ -153,6 +153,41 @@ function update($app, $global, $quiet = $false, $independent, $suggested, $use_c
 
     write-host "Updating '$app' ($old_version -> $version)"
 
+    # region Workaround
+    # Workaround for https://github.com/lukesampson/scoop/issues/2220 until install is refactored
+    # Remove whole region after proper fix
+    Write-Host "Downloading new version to verify integrity of files"
+    if (aria2_enabled) {
+        dl_with_cache_aria2 $app $version $manifest $architecture $cachedir $manifest.cookies $true $check_hash
+    } else {
+        # dl_with_cache $app $version $
+        $urls = url $manifest $architecture
+
+        foreach ($url in $urls) {
+            dl_with_cache $app $version $url $null $manifest.cookies $true
+
+            $manifest_hash = hash_for_url $manifest $url $architecture
+            $source = fullpath (cache_path $app $version $url)
+            $ok, $err = check_hash $source $manifest_hash $(show_app $app $bucket)
```

Comment:
The workaround always check hash even if the user specifies `--skip` or install a [`nightly` version of app](https://github.com/lukesampson/scoop/blob/0156d5d10be1827de900637de454531a143bf43a/libexec/scoop-update.ps1#L135-L138). You should respect users' behavior if they want to do that, [the option of just downloading packages](https://github.com/lukesampson/scoop/blob/9ce2fda8543913bc229203feffcbac5232ff266d/lib/install.ps1#L319) before uninstalling, but without hash checking.